app/models/actor.rb

```diff
@@ -0,0 +1,13 @@
+class Actor < ApplicationRecord
+  has_many :actor_movies
```

Comment:
Nice M2M set up! Syntax errors with db associations/setup can be hard so I suggest keeping a cheatsheet with what you did to get this working because it looks good!